alerting/src/main/kotlin/com/amazon/opendistroforelasticsearch/alerting/MonitorRunner.kt

```diff
@@ -93,74 +98,77 @@ class MonitorRunner(
     private val xContentRegistry: NamedXContentRegistry,
     private val alertIndices: AlertIndices,
     clusterService: ClusterService
-) : JobRunner {
+) : JobRunner, CoroutineScope, AbstractLifecycleComponent(settings) {
 
-    private val logger = ElasticAPI.INSTANCE.getLogger(MonitorRunner::class.java, settings)
+    private lateinit var job: Job
```

Comment:
As discussed offline: this job and the scheduled job (in runJob) both being referred to as job is confusing. Let's be more explicit with naming.